src/jit/SimdInl.h

```diff
@@ -311,3 +311,25 @@ static void emitGlobalSet128(sljit_compiler* compiler, Instruction* instr)
     sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_MEM1(SLJIT_R0), context->globalsStart + globalSet->index() * sizeof(void*));
     sljit_emit_simd_mov(compiler, SLJIT_SIMD_STORE | SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_128, src.arg, SLJIT_MEM1(SLJIT_R0), JITFieldAccessor::globalValueOffset());
 }
+
+static void emitSelect128(sljit_compiler* compiler, Instruction* instr)
+{
+    Operand* operands = instr->operands();
+    assert(instr->opcode() == ByteCode::SelectOpcode && instr->paramCount() == 3);
+
+    JITArg args[2];
+    JITArg target(operands + 3);
+    JITArg cond(operands + 2);
+    simdOperandToArg(compiler, operands, args[0], SLJIT_SIMD_ELEM_128, SLJIT_FR2);
+    simdOperandToArg(compiler, operands + 1, args[1], SLJIT_SIMD_ELEM_128, SLJIT_FR1);
+    sljit_s32 type = SLJIT_SIMD_REG_128;
+    if (SLJIT_IS_MEM(target.arg)) {
+        type |= SLJIT_SIMD_STORE | SLJIT_SIMD_ELEM_128;
```

Comment:
The elem is 128 bit even in the load case.

Reply:
Done in current version.